docs/docs/Tutorials/Install/Install_source.md

```diff
@@ -47,15 +47,21 @@ Use Ubuntu 22.04.
         export GZ_VERSION=garden
         ```
 
-4. Install ROS dependencies
+4. Set `SDF_PATH` to allow `robot_state_publisher` parse the robot description
+   from the sdformat model (place this in ~/.bashrc for convenience if rebuilding often):
+       ```
+       export SDF_PATH=$GZ_SIM_RESOURCE_PATH
```

Comment:
Same as comment above. I think this needs to go after sourcing `install/setup.sh` (or `install/setup.bash`)